cmd/osbuild-composer/composer.go

```diff
@@ -99,15 +100,17 @@ func (c *Composer) InitWeldr(repoPaths []string, weldrListener net.Listener) err
 		name = "centos-stream-8"
 	}
 
-	repos, err := rpmmd.LoadRepositories(repoPaths, name)
+	rr, err := reporegistry.New(repoPaths)
```

Comment:
hmm, `name` is no longer used. This needs to be addressed. Not sure how exactly though...

Reply:
Yes, it's on my TODO for today. I'll have to move the "name mangling" to the Weldr API, similar as @bcl did it in his PR. I'll probably just copy his change 1:1 (Thanks @bcl :-D)

The thing is that the `Name()` method of the distro instance passed to Weldr API won't return the expected name, so Weldr would not be able to get the hostDistro repositories from the RepoRegistry.

Reply:
This is fixed by https://github.com/osbuild/osbuild-composer/pull/1387/commits/4885e2d3a728fd91615eed991d569d89d005e1bb